libexec/scoop-status.ps1

```diff
@@ -98,8 +98,12 @@ if($missing_deps) {
     }
 }
 
-if(!$old -and !$removed -and !$failed -and !$missing_deps) {
+if(!$old -and !$removed -and !$failed -and !$missing_deps -and !$needs_update) {
     success "Everything is ok!"
 }
 
+if ($needs_update) {
+    warn "Cannot check for app updates while Scoop is out of date."
```

Comment:
If Scoop is out of date and there are app updates too, these changes will make Scoop print that it can't check for app updates right after it has printed out what the app updates are.

If you were meaning to prevent Scoop checking for app updates if it itself is out of date — that's a judgement call that I don't agree with. Scoop is updated many times a day and is almost guaranteed to be out of date. It's still useful to print the apps it knows to be out of date.

You're right about the 'Everything is ok!' message being confusing.